.github/workflows/publish.yml

```diff
@@ -0,0 +1,671 @@
+name: Publish
+
+on:
+  workflow_call:
+    inputs:
+      git_ref:
+        type: string
+        required: true
+      mode:
+        type: choice
+        required: true
+        options:
+          - nightly
+          - release
+      debug_symbols:
+        type: boolean
+        required: false
+        default: false
```

Comment:
This controls whether we publish an extra binary with debug symbols, right? I'd be ok with doing this for both nightly and real releases to remove one extra difference.

Reply:
It does, yes.

Generally we've strayed away, I believe, from including debug symbols because of the resulting binary size. Let me do a release build with and without debug symbols and report back what the difference is.